frontend/src/components/PopupTabs/ShareSequencePopupTab.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <ContainerComponent>
+    <h1>
+      Geben Sie die ID der Lehrkraft ein, mit der die Sequenz geteilt werden
```

Comment:
This should be per name or mail since the user cant see the ID

Reply:
Can there be 2 users with the same username?

Reply:
no this is blocked in the account controller, mail and name are unique